exampleCourse/questions/element/orderBlocks/question.html

```diff
@@ -75,11 +76,12 @@
 >
   <pl-answer correct="true" ranking="1" indent="0">def my_sum(first, second):</pl-answer>
   <pl-answer correct="true" ranking="2" indent="1">sum = 0</pl-answer>
-  <pl-answer correct="true" ranking="3" indent="1">sum += first</pl-answer>
-  <pl-answer correct="true" ranking="3" indent="1">sum += second</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="1" tag="addfirst">sum += first</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="1" tag="addsecond">sum += second</pl-answer>
   <pl-answer correct="true" ranking="4" indent="1">return sum</pl-answer>
-  <pl-answer correct="false">sum + first</pl-answer>
-  <pl-answer correct="false">sum + second</pl-answer>
+  <pl-answer correct="false" distractor-for="addfirst">sum + first</pl-answer>
+  <pl-answer correct="false" distractor-for="addfirst">sum = first</pl-answer>
+  <pl-answer correct="false" distractor-for="addsecond">sum + second</pl-answer>
```

Comment:
It seems like `correct="true" distractor-for="..."` would be an invalid configuration, but AFAICT we don't have validation for that. Did you consider validating that during `prepare()`?

Reply:
Nice catch! I will add validation for that.